app/src/main/java/com/cornellappdev/uplift/ui/components/general/CalendarBar.kt

```diff
@@ -56,13 +67,54 @@ fun CalendarBar(selectedDay: Int, daysAhead: Int = 14, onDaySelected: (Int) -> U
     }
 }
 
+/**
+ * A scrollable row of days of the week starting from the current day. Calls [onDaySelected] with
+ * the day of the week selected.
+ */
+@Composable
+fun DayOfWeekBar(
+    initialSelectedDays: Set<String> = emptySet(),
+    onDaySelected: (Set<String>) -> Unit
+) {
+    var selectedDays by remember { mutableStateOf(initialSelectedDays) }
+
+    val daysOfWeek = listOf("M", "T", "W", "Th", "F", "Sa", "Su")
+
```

Comment:
Okay ya similar idea here--please shift over to having no internal state! Sorry to promote all these changes, but changes like this improve syncing ability between UI & VM and make UI implementation easier :-)